textBasedAdventure.js

```diff
@@ -1,5 +1,140 @@
-const readline1 = require('readline-sync')
+const readlineSync = require('readline-sync');
+const chalk = require('chalk');
+//Game Start
+const gameStart = () => {
+console.log(chalk.blue('Hello user '));
+console.log(chalk.blueBright('This Game is simple '));
+console.log(chalk.blue('Just a yes no maybe so '));
+console.log(chalk.blueBright('With a bit of numbers'));
+console.log(chalk.green('Enjoy!'));
+let nameInput = readlineSync.question('Please enter your name: ');
+console.log(chalk.yellow(`Hello ${nameInput}!  Welcome to my game.`))
+gameStart2()
+gameLoop()
+}
+//Game loop
+const gameLoop = () => {
+    let tryCount = 3;
+    for(i = tryCount; i !== 0; i--) {
+        if(tryCount > 3) {
+            gameLoop()
+            
+        }else{
+            tryCount --;
+            console.log("Number of remaing attempts left " + tryCount)
+            if(tryCount === 0) {
+                gameFirst();
+            }
+            break;
+        }
+    }
+}
```

Comment:
rethink this for loop, if the count is greater than 3 this function is going to call itself. But, it will never land inside if statement because 3 is not greater than 3. Think out what this function should do before rewriting it